example/src/scenarios/get-pid.tsx

```diff
@@ -3,12 +3,31 @@ import {
   PID,
   createCryptoContextFor,
   getCredentialIssuerEntityConfiguration,
+  getTrustAnchorEntityConfiguration,
+  verifyTrustChain,
+  renewTrustChain,
 } from "@pagopa/io-react-native-wallet";
 import { error, result, toResultOrReject } from "./types";
 import getWalletInstanceAttestation from "./get-attestation";
 
 const walletProviderBaseUrl = "https://io-d-wallet-it.azurewebsites.net";
 const pidProviderBaseUrl = "https://api.eudi-wallet-it-pid-provider.it/ci";
+const trustAnchorBaseUrl =
+  "https://demo.federation.eudi.wallet.developers.italia.it/";
+
+async function trust(trustChain: string[]): Promise<void> {
+  const trustAnchorEntity = await getTrustAnchorEntityConfiguration(
+    trustAnchorBaseUrl
+  );
+
+  // test verify trust chain
+  await verifyTrustChain(trustAnchorEntity, trustChain);
+
+  // test renew of the trust chain
+  // (just for test, needed only when input chain fails to validate)
+  const renewedChain = await renewTrustChain(trustChain);
+  await verifyTrustChain(trustAnchorEntity, renewedChain);
```

Comment:
What do you think about adding check here if the trust chain is expired? Or maybe add a utility function so that it can be used easily?

Reply:
You mean something like
```ts
async function trust(trustAnchorEntity, chain){
  try {
    return verifyTrustChain(trustAnchorEntity, chain) 
  } catch(ex){
    const renewedChain = await renewTrustChain(chain);
    return verifyTrustChain(trustAnchorEntity, renewedChain);
  }
}
```
Sure It will be useful, we can do it.

I'm not sure if we want to discriminate whether the trust chain verification fails for expiration or something else. I thought we could just try again with a renewed chain on every failure.

Reply:
Yep. I would only throw an exception if renewal fails or post renewal verification fails.

Reply:
It will, as both `renewTrustChain` and `verifyTrustChain` may throw an exception. Do you mean a more descriptive error? Sounds resonable.

Reply:
I proposed another interface [here](https://github.com/pagopa/io-react-native-wallet/pull/52/commits/be1ddbe72fcc8c7015a832063d52a608399d312f)